pytest.ini

```diff
@@ -1,3 +1,5 @@
 [pytest]
 norecursedirs=listenbrainz/tests/integration listenbrainz_spark
 addopts = --cov=listenbrainz
+
+testpaths = listenbrainz/listenstore/tests/test_timescalelistenstore.py
```

Comment:
It should test this path automatically, not sure why we need this line

Reply:
I am running tests in only 1 file